services/user.service.js

```diff
@@ -37,20 +37,20 @@ class UserService extends BaseService {
   }
 
   async doList() {
-    let allUsers = [];
-    let response;
+    let allUsers = []
+    let response
 
     try {
-      let userRefSnapshot = await this.collection.get();
+      let userRefSnapshot = await this.collection.get()
 
-      userRefSnapshot.forEach((doc) => {
-        let userData = doc.data();
-        userData.id = doc.id;
-        allUsers.push(userData);
-      });
+      userRefSnapshot.forEach(doc => {
+        let userData = doc.data()
+        userData.id = doc.id
+        allUsers.push(userData)
```

Comment:
i think we can refactor this to this 
```suggestion
        allUsers.push(doc.data())
```

Reply:
Jose, all this is based on firebase connection, all should be refactored for mongo indeed. including your suggestion, but is not the current Sprint goal, can we solve this and all changes not related to the event user and event validation as a refactor in backlog?

Reply:
Thats ok, but actually making this change in particular wont take you a lot of time from my perspective, or will it ?

Reply:
Is not about time, is about the scope, but I can do it.